code/datums/gamemode/dynamic/dynamic_rulesets_roundstart.dm

```diff
@@ -50,18 +50,21 @@
 //               CHANGELINGS                ////////////////////////////////////////////////////////////////////////////////////////////////////////////////////////////
 //                                          //
 //////////////////////////////////////////////
-/*
+
+
+//note: this can only fire on snowmap
+
 /datum/dynamic_ruleset/roundstart/changeling
 	name = "Changelings"
 	role_category = CHANGELING
-	protected_from_jobs = list("Security Officer", "Warden", "Merchant", "Head of Personnel", "Detective", "Head of Security", "Captain")
-	restricted_from_jobs = list("AI","Cyborg","Mobile MMI")
-	enemy_jobs = list("Security Officer","Detective","Head of Security", "Captain")
-	required_enemies = list(1,1,0,0,0,0,0,0,0,0)
+	protected_from_jobs = list("Security Officer", "Warden","Merchant", "Head of Personnel", "Detective", "Head of Security", "Captain")
+	restricted_from_jobs = list("AI","Cyborg","Mobile MMI", "Chaplain")
```

Comment:
:thinking: what the actual fuck

Reply:
G*d wouldn't let His chaplain be impersonated